src/node.cc

```diff
@@ -1513,7 +1513,14 @@ static void ReportException(Environment* env,
         name->IsUndefined()) {
       // Not an error object. Just print as-is.
       node::Utf8Value message(env->isolate(), er);
-      PrintErrorString("%s\n", *message);
+
+      // If an error occurs while creating message, the length will be zero,
+      // but PrintErrorString() will display garbage characters. So, if the
+      // length is zero, just print a newline.
+      if (message.length() == 0)
+        PrintErrorString("\n");
+      else
+        PrintErrorString("%s\n", *message);
```

Comment:
> I would print a message like `"<toString() threw exception>"` instead of a blank line.

I thought about that, but the current check is insufficient, since empty string is a valid `toString()` response. Would you be OK with me using a `TryCatch` to actually detect if an error was thrown?